File: src/algorithms/smallest-even-multiple.js
/**
 * Given a positive integer n, returns the smallest positive integer that is a multiple of both 2 and n.
 * 
 * @param {number} n - A positive number
 * @returns {number} - Returns a positive number
 */
function smallestEvenMultiple (n) {
  return n % 2 === 0 ? n : n * 2;
}

File: src/algorithms/smallest-even-multiple.py
def smallestEvenMultiple(n):
    """Given a positive integer n, return the smallest positive integer that is 
    a multiple of both 2 and n.

    Args:
        n (int): A positive integer.

    Returns:
        int: Returns the smallest positive integer that is a mult of 2 and n.
    """
    return n if n % 2 == 0 else n * 2
